### PR TITLE
Fix class control to correctly add a new semester

### LINKED ISSUE #153: Fix attendance
Attendance is currently broken on live site. (more info to come)

### PR DESCRIPTION
**Pull request by kmcnellis**
working towards #138, fixes #153

### CODE REVIEW

File: client/app/admin/classcontrol/classcontrol.controller.js [hunk 1]
```diff
@@ -48,6 +49,54 @@ angular.module('observatory3App')
...
+          semester: thisSemester
+        }).success(function(){
+            updateClassYear();
+            $scope.newSemester = {year:"", semester:""};
+            $scope.showNewSemester = false;
+          })
+          .error(function(err){
+            console.error("Error creating new semester", err);
+          });
+
```
Comment (agundy) [inline client/app/admin/classcontrol/classcontrol.controller.js]: Extra line?

File: client/app/admin/classcontrol/classcontrol.html [hunk 2]
```diff
@@ -1,29 +1,56 @@
...
-    <button ng-show='attendanceCode' ng-click='showAttendanceCode=true' class='btn btn-primary'>Show Attendance Code</button>
-    <div class='big-attendance' ng-show='showAttendanceCode' ng-click='showAttendanceCode=false;'>
-      <div class='small-text'>
-        The Attendance Code is...
-      </div>
-      <div class='big-text'>
-        {{ attendanceCode }}
+  <div class="row">
+    <div class="col-sm-12">
+
```
Comment (agundy) [inline client/app/admin/classcontrol/classcontrol.html]: whitespace

File: client/app/admin/classcontrol/classcontrol.html [hunk 3]
```diff
@@ -1,29 +1,56 @@
...
+              <div class="col-sm-12">
+
+              <div class='small-text'>
+                The Attendance Code is...
+              </div>
+              <div class='big-text'>
+                {{ attendanceCode }}
+              </div>
+            </div>
+
```
Comment (agundy) [inline client/app/admin/classcontrol/classcontrol.html]: extra return?

File: client/app/admin/classcontrol/classcontrol.html [hunk 4]
```diff
@@ -1,29 +1,56 @@
...
-  <div class='newSemester'>
-    <div class="input-group">
-        <span class="input-group-btn">
-          <button class="btn btn-default" type="button">Create new semester</button>
-        </span>
-        <input type="text" class="form-control" placeholder="2014fall">
+
+  <div class="row">
+    <div class="col-sm-12">
+
```
Comment (agundy) [inline client/app/admin/classcontrol/classcontrol.html]: I don't think we need this return

File: client/app/admin/classcontrol/classcontrol.controller.js (hunk 1 above)
Comment (kmcnellis) [inline client/app/admin/classcontrol/classcontrol.controller.js]: What line?

File: client/app/admin/classcontrol/classcontrol.html (hunk 2 above)
Comment (kmcnellis) [inline client/app/admin/classcontrol/classcontrol.html]: Specify where

File: client/app/admin/classcontrol/classcontrol.html (hunk 3 above)
Comment (kmcnellis) [inline client/app/admin/classcontrol/classcontrol.html]: Specify where

File: client/app/admin/classcontrol/classcontrol.html (hunk 4 above)
Comment (kmcnellis) [inline client/app/admin/classcontrol/classcontrol.html]: Why?

File: client/app/admin/classcontrol/classcontrol.controller.js [hunk 5]
```diff
@@ -48,6 +49,54 @@ angular.module('observatory3App')
...
+      if(form.$valid) {
+        $http.post("/api/classyear/", {
+          semester: thisSemester
+        }).success(function(){
+            updateClassYear();
+            $scope.newSemester = {year:"", semester:""};
+            $scope.showNewSemester = false;
+          })
+          .error(function(err){
+            console.error("Error creating new semester", err);
```
Comment (agundy) [inline client/app/admin/classcontrol/classcontrol.controller.js]: i think we should silence this for now but in the future log js errors to the server

File: client/app/admin/classcontrol/classcontrol.controller.js (hunk 1 above)
Comment (agundy) [inline client/app/admin/classcontrol/classcontrol.controller.js]: line 94

File: client/app/admin/classcontrol/classcontrol.html (hunk 2 above)
Comment (agundy) [inline client/app/admin/classcontrol/classcontrol.html]: line 6

**Pull request merged**